adoc/chapters/programming_interface.adoc

```diff
@@ -14622,6 +14622,15 @@ interoperability member functions provided by the [code]#interop_handle# class.
 
 Local <<accessor,accessors>> cannot be used within a <<host-task>>.
 
+[NOTE]
+====
+If a C++ lambda is passed to a <<host-task>>, the lambda may capture by
+reference or by value.
+Since the <<host-task>> callable executes asynchronously, care must be taken to
+ensure that lifetimes of objects captured by reference by a <<host-task>> lambda
```

Comment:
Even for objects captured by copy you still need to unsure that it works under the hood with the program logic.
The trivial example is a `std::ref` or a pointer captured by copy. ;-)